src/lesson_5/script.js

```diff
@@ -0,0 +1,85 @@
+for (let i = 10; i <= 25; i +=0.5) {
+    console.log("i is equal to =", i);
+}
+
+const userNumber = prompt("Напишіть будь яке число");
+console.log("userNumber =", userNumber);
+
+let isPrime = "prime"; 
+
+for (var i = 2; i < userNumber; i++) {
```

Comment:
```suggestion
for (let i = 2; i < userNumber; i++) {
```
The common practice is to use let/const for variable initialization.
`let` and `const` are not just new cool keywords added to JS for fun, they also introduce block scope which allows us to write cleaner and less error-prone code.